packages/xo-server/src/index.js

```diff
@@ -125,10 +125,14 @@ async function setUpPassport(express, xo, { authentication: authCfg }) {
     { label = strategy.label, name = strategy.name } = {}
   ) => {
     passport.use(name, strategy)
-
```

Comment:
Please remove the unrelated change.

Reply:
This is related as I wanted to have two similar blocks: one to add an one to remove the strategy.